vagrant/base.sh

```diff
@@ -64,15 +64,20 @@ get_kafka() {
 
     kafka_dir=/opt/kafka-$version
     url=https://s3-us-west-2.amazonaws.com/kafka-packages-$version/kafka_$scala_version-$version.tgz
+    # the .tgz above does not include the streams test jar hence we need to get it separately
+    url_streams_test=https://s3-us-west-2.amazonaws.com/kafka-packages-$version/kafka-streams-$version-test.jar
```

Comment:
I'll open a separate JIRA for a releaseTestTarGzAll for now. The test will be the same either way and i'll redo base.sh then. cc @dguy .